subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -845,6 +845,66 @@
     return transform("transform: " + _taskDescriptor.getDescription(func.getClass().getName()), func);
   }
 
+  /**
+   * Create a new task that will transform the result of this task.
+   * Returned task will complete with value calculated by a task returned by the function.
+   *
+   * This is similar to {@link #transform(String, Function1)} except the transformation is done by executing the task returned
+   * by the function.
```

Comment:
can we have a tracevis flow explanation to this method like the one for existing `transform` API? As well as some code example usage to demonstrate why this new API is useful?

Reply:
added